src/harness/testcases/WINNF_FT_S_SCS_testcase.py

```diff
@@ -64,3 +66,236 @@ def test_WINNF_FT_S_SCS_5(self):
     Checks that a CBSD registration with this configuration succeed.
     """
     self.doTestCipher('ECDHE-RSA-AES128-GCM-SHA256')
+
+  @winnforum_testcase
+  def test_WINNF_FT_S_SCS_6(self):
+    """Unrecognized root of trust certificate presented during registration.
+    Checks that SAS UUT response with fatal alert with unknown_ca.
+    """
+    device_cert = os.path.join('certs', 'unrecognized_device.cert')
+    device_key = os.path.join('certs', 'unrecognized_device.key')
+    self.assertTlsHandshakeFailure(device_cert, device_key)
+
+  @winnforum_testcase
+  def test_WINNF_FT_S_SCS_7(self):
+    """Corrupted certificate presented during registration.
+    Checks that SAS UUT response with fatal alert message.
+    """
+    with open(os.path.join('certs', 'client.cert')) as f:
+      data = "".join(f.read().split('-----END CERTIFICATE-----'))
+    def corrupt_signature(cert, offset):
+      temp = list(cert)
+      temp[len(cert) - offset] = chr(ord(temp[len(cert) - offset ]) + 1)
+      return ''.join(temp).strip() + '\n-----END CERTIFICATE-----\n'
+    with open(os.path.join('certs', 'corrupted_client.cert'),mode='w') as f:
+      f.write(corrupt_signature(data, 10))
+
+    device_cert = os.path.join('certs', 'corrupted_client.cert')
+    device_key = os.path.join('certs', 'client.key')
+    self.assertTlsHandshakeFailure(device_cert, device_key)
+
+  @winnforum_testcase
+  def test_WINNF_FT_S_SCS_8(self):
+    """Self-signed certificate presented during registration.
+    Checks that SAS UUT response with fatal alert message.
+    """
+    device_cert = os.path.join('certs', 'self_signed_client.cert')
+    device_key = os.path.join('certs', 'client.key')
+    self.assertTlsHandshakeFailure(device_cert, device_key)
+
+  @winnforum_testcase
+  def test_WINNF_FT_S_SCS_9(self):
+    """Non-CBRS trust root signed certificate presented during registration.
+    Checks that SAS UUT response with fatal alert message.
+    """
+    device_cert = os.path.join('certs', 'non_cbrs_signed_device.cert')
+    device_key = os.path.join('certs', 'non_cbrs_signed_device.key')
+    self.assertTlsHandshakeFailure(device_cert, device_key)
+
+  @winnforum_testcase
+  def test_WINNF_FT_S_SCS_10(self):
+    """Certificate of wrong type presented during registration.
+    Checks that SAS UUT response with fatal alert message.
+    """
+    device_cert = os.path.join('certs', 'sas_ca_signed_client.cert')
+    device_key = os.path.join('certs', 'client.key')
+    self.assertTlsHandshakeFailure(device_cert, device_key)
+
+  @winnforum_testcase 
+  def test_WINNF_FT_S_SCS_11(self):
+    """Blacklisted Certificate presented during registration.
+    Checks that SAS UUT response with fatal alert message.
+    """
+    #Blacklist certificate
+    device_a = json.load(open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    self._sas_admin.BlacklistByFccIdAndSerialNumber({'fccId': device_a['fccId'],'CbsdSerialNumber': device_a['cbsdSerialNumber']})
```

Comment:
We need to actually blacklist the *certificate*, not the FCC ID + SN.

Reply:
will check this with the developer and get back to you soon.

Reply:
We have removed the test cases SCS 11,17,18 and 19 from this PR. They will be submitted separately once the approach for blacklisting has been finalized.